### DISCUSSION TITLE
Status Update: Paper publication 2023-09-15

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/ACE_Framework#1] [Announcements]
Update notes for everyone:

1. The academic conference paper has been submitted. 
2. Initial draft of this repo is here, still need to add more prompt examples and diagrams.
3. I have potential collaborations coming up to create hands on implementations.

**Comment by daveshap** [comment]
I've updated the Contributions.md to be more diplomatic and specific about how to participate

**Comment by DataBassGit** [comment]
Are we going to publish implementations in this repo or a separate one linked from here? Collaborators will be best facilitated by having a repo (Dev branch) to publish to for best collaboration workflow and version control.

**Reply by lancecarlson** [reply]
@DataBassGit I think we should have a separate implementation repo

**Reply by daveshap** [reply]
I'd like to have at least one toy demo in this repo. Probably langchain or even maybe native Python. Just something that people can easily spin up and see that yes, it works.

**Reply by DataBassGit** [reply]
We've developed AgentForge independently frrom langchain. However, our database implementation is very heavy due to being local. Still I think AF can support all of the features necessary for the ACE Framework. And because of how simple agent implementation is, adding a player folder to this repo should be incredibly straight forward. I know Lance sent a pull to the repo a few months ago, but we've come a LONG way since then.
